src/store/lokapi.ts

```diff
@@ -93,14 +93,32 @@ export var moduleLokAPI = {
       let accounts: any;
       try {
         accounts = await lokApiService.getAccounts();
-        let balance = await accounts[0].getBalance();
-        let symbol = await accounts[0].getSymbol();
-        state.bal = balance;
-        state.curr = symbol;
+        if (accounts.length > 0) {
+          // Compute a global moneyAccount balance
+          Promise.allSettled(accounts.map((a:any) => {
+            return a.getBalance()
+          }))
+          .then((balances:any) => {
+            state.bal = balances.reduce((s:number, b:any) => {
+              return s + parseFloat(b.value)
```

Comment:
using floats is kinda risky if you don't round at the end to the decimal places wanted. While testing your code, I got this displayed :
"97.38000000000001 mixed currencies"

For now, you could use `.toFixed(2)`. Later, we'll ask how many digits to the currency itself (this info exists on cyclos at least).

Reply:
Allright. Got it.